docs/modules/hive/partials/supported-versions.adoc

```diff
@@ -2,17 +2,19 @@
 // This is a separate file, since it is used by both the direct Hive-Operator documentation, and the overarching
 // Stackable Platform documentation.
 
-- 4.0.1 (experimental)
-- 4.0.0 (LTS)
+- 4.0.1 (LTS)
+- 4.0.0 (deprecated)
 - 3.1.3 (deprecated)
 
 === Hive 4 issues
 
-As of SDP 25.7, Hive 4 is still relatively new and there are some known issues, especially when using it with Iceberg and Trino.
-The missing comppatibilty with Iceberg also means that Spark jobs using this storage may fail.
+Hive 4 has known compatibility issues, especially when using it with Iceberg and Trino.
+The missing compatibility with Iceberg also means that Spark jobs using this storage may fail.
+
+**Workaround:** If you encounter issues with Hive 4, use Hive 3.1.3 instead until these upstream issues are resolved.
```

Comment:
As mentioned in https://github.com/stackabletech/hive-operator/issues/626#issuecomment-3273914299 a downgrade story would be great